Source/Engine/Physics/PhysX/PhysicsBackendPhysX.cpp

```diff
@@ -976,6 +984,39 @@ void PhysicalMaterial::UpdatePhysicsMaterial()
 
 bool CollisionCooking::CookConvexMesh(CookingInput& input, BytesContainer& output)
 {
+#if 0 // data extractor
+
+    String out = String::Format(L"\n static Float3* points [{0}] = ", input.VertexCount);
+    out.Append('{');
+    auto data = Array<Float3>(input.VertexData, input.VertexCount);
+    for (size_t i = 0; i < data.Count(); i++)
+    {
+        auto d = data[i];
+        out.Append(String::Format(L"\tFloat3({0},{1},{2}),\n", d.X, d.Y, d.Z));
+    }
+    out.Append(L"};");
+    if (input.Is16bitIndexData)
+    {
+        auto data2 = Array<PxU16>((PxU16*)input.IndexData, input.IndexCount);
+        for (size_t i = 0; i < data2.Count(); i++)
+        {
+            auto d = data2[i];
+            out.Append(String::Format(L"{0},\n",d));
+        }
+    }
+    else
+    {
+        auto data2 = Array<PxU32>((PxU32*)input.IndexData, input.IndexCount);
+        for (size_t i = 0; i < data2.Count(); i++)
+        {
+            auto d = data2[i];
+            out.Append(String::Format(L"{0},\n", d));
+        }
+    }
+
+    LOG_STR(Info, out.ToString());
+#endif
+
```

Comment:
I assume this is not used? does it need to stay? maybe for debug purposes?

Reply:
> I assume this is not used?

yes it is unused

> does it need to stay?

idk i used it to extract verts for breaking and just left it there

> maybe for debug purposes?
if somfing break it will have use 
note is striped from compilation any way